modules/core/test/v2/unit/coins/stx.ts

```diff
@@ -102,10 +94,10 @@ describe('STX:', function () {
     goodAddresses.map(addr => { basecoin.isValidAddress(addr).should.equal(true); });
   }));
 
-  it('should verify addresses', (function () {
-    goodAddresses.map(address => { basecoin.verifyAddress({ address }).should.equal(true); });
```

Comment:
Same question here.

Reply:
in this case here we actually have tests for `isValidAddress` a few lines above